generators/chipyard/src/main/scala/IOBinders.scala

```diff
@@ -249,11 +258,11 @@ class WithBlockDeviceModel extends OverrideIOBinder({
 })
 
 class WithLoopbackNIC extends OverrideIOBinder({
-  (system: CanHavePeripheryIceNICModuleImp) => system.connectNicLoopback(); Nil
+  (system: CanHavePeripheryIceNICModuleImp, p) => system.connectNicLoopback(); Nil
 })
 
 class WithSimNIC extends OverrideIOBinder({
-  (system: CanHavePeripheryIceNICModuleImp) => system.connectSimNetwork(system.clock, system.reset.asBool); Nil
+  (system: CanHavePeripheryIceNICModuleImp, p) => system.connectSimNetwork(system.clock, system.reset.asBool); Nil
 })
 
 // Note: The parameters instance is accessible only through the BaseSubsystem
```

Comment:
Delete this comment.

Reply:
Moved the comment to where we match for `LazyModule` or `LazyModuleImpLike`